modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -103,6 +103,8 @@ module LocaleModelHelpSetup {
     here.runningTaskCntSet(0);  // locale init parallelism mis-sets this
   }
 
+  //TODO: Create a helpSetupRootLocale function for GPUs
```

Comment:
Is this something that still needs to be done in the future?  It's fine if the answer is "yes", just checking that it's not a leftover from earlier that can be removed now.